finance/migrations/0009_auto_20160720_0725.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('finance', '0008_auto_20160320_2349'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='reportingperiod',
+            name='form',
+        ),
+        migrations.RemoveField(
+            model_name='reportingperiod',
+            name='locality',
+        ),
+        migrations.AlterModelOptions(
+            name='independentmoney',
+            options={'ordering': ('-beneficiary__ballot_item_selection__ballot_item__ballot__date', '-report_date'), 'verbose_name_plural': 'independent money'},
+        ),
+        migrations.RemoveField(
+            model_name='independentmoney',
+            name='reporting_period',
+        ),
+        migrations.DeleteModel(
+            name='ReportingPeriod',
+        ),
```

Comment:
The point of migrations is to record historical database state and allow incremental schema changes to happen simultaneously between developers (and also in production).

Since we're all so used to it at this point, it's not the biggest deal, but we all will have to do (as Aaron points out):
```
python manage.py reset_db
python manage.py setuptestserver
```

to recover.